src/null-safety/index.md

```diff
@@ -78,11 +78,11 @@ Dart null safety support is based on the following three core design principles:
 
 ## Dart 3 and null safety
 
-Dart 3---planned for a mid-2023 release---always has sound null safety.
-Dart 3 will prevent code from running without it.
+Dart 3 always has sound null safety.
```

Comment:
The term we've been using in the other PRs is ""Dart 3 has null safety built-in"